members-list.md

```diff
@@ -17,6 +17,10 @@ Also don't be a cop-out by saying something like "I am happy/looking forward to
 
 ## The list 
 [Add your details here]`<--Don't remove this line`<br/>
+**Name**:  Ishaan Shettigar  <br/>
+**SRN**:  PES1UG20CS170  <br/>
+**Link to Profile**:  [IshaanShettigar]https://github.com/IshaanShettigar  <br/>
```

Comment:
```suggestion
**Link to Profile**:  [IshaanShettigar](https://github.com/IshaanShettigar)  <br/>
```

You need to add brackets around the URL for it to show up like a link. ([refrence](https://www.markdownguide.org/basic-syntax/#links))

(You can also preview the file by opening it [here](https://github.com/IshaanShettigar/members-list/blob/master/members-list.md) in your fork. Currently you can see that the link doesn't look good.)

Ask on the slack if you need help with anything :)